gather_vars

```diff
@@ -0,0 +1,11 @@
+extra_args=''
+
+if [ "$1" == "--rcm" ]; then
+  shift
+  extra_args+='-e @../../rcm_vars.yml'
```

Comment:
Wondering if this might be a problem running from ec2 directory, assuming this was written for sourcing from "local/linux" or "local/mac" instead of "ec2"